tests/manage/mcg/test_bucket_creation.py

```diff
@@ -33,7 +33,14 @@ class TestBucketCreation(MCGTest):
         argvalues=[
             pytest.param(
                 *[3, "S3", None],
-                marks=[pytest.mark.polarion_id("OCS-1298"), tier1, acceptance],
+                marks=[
+                    pytest.mark.polarion_id("OCS-1298"),
+                    tier1,
+                    acceptance,
+                    pytest.mark.skip(
```

Comment:
Shouldn't we use bugzilla decorator to check directly against the bug?
https://ocs-ci.readthedocs.io/en/latest/writing_tests.html#bugzilla-decorator

Reply:
done